lib/urlwatch/filters.py

```diff
@@ -155,6 +156,30 @@ def filter(self, data, subfilter=None):
             return data
 
 
+class BeautifyFilter(FilterBase):
+    import jsbeautifier
```

Comment:
It does not make sense to do this in the class context, you have to do it inside the function code (inside the block that starts with `def filter(...):`